tests/test_semver.py

```diff
@@ -80,3 +80,18 @@ def test_should_versioninfo_isvalid():
 def test_versioninfo_compare_should_raise_when_passed_invalid_value():
     with pytest.raises(TypeError):
         Version(1, 2, 3).compare(4)
+
+
+@pytest.mark.parametrize(
+    "caller, callee",
```

Comment:
💡This is entirely personal preference but I don't like pytest doing string operations 😶
```suggestion
    ("caller", "callee"),
```

Reply:
That would be inconsistent with the other tests. You are right, but I like consistency.